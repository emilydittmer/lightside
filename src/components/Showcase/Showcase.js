import React from 'react';
import './Showcase.scss';

const Showcase = ({films}) => {
  let movie = films[0]
  console.log(movie)
  return (
    <section>
      <p>{movie.openingCrawl}</p>
      <h3>{movie.title}</h3>
      <h4>{movie.releaseDate}</h4>
    </section>
  )
}

export default Showcase;