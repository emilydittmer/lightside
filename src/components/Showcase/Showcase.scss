@import '../../sass/_mixins.scss';
@import '../../sass/_variables.scss';
