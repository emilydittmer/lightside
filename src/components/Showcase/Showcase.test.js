import React from "react";
import { shallow } from "enzyme";
import Showcase from "./Showcase";

describe("Showcase", () => {
  it("should expect wrapper to match snapshot", () => {
    let wrapper = shallow(
      <Showcase films={[{ openingCrawl: "random testing sentence." }]} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
