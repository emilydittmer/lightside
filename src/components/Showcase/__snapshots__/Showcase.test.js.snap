// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Showcase should expect wrapper to match snapshot 1`] = `ShallowWrapper {}`;
